ui/app/mirrors/create/helpers/qrep.ts

```diff
@@ -28,15 +28,15 @@ export const qrepSettings: MirrorSetting[] = [
     required: true,
   },
   {
-    label: 'Create Destination Table',
+    label: 'Create Watermark Table On Destination',
```

Comment:
watermark table is confusing right Amogh? that's why we wanted it as destination table.

Reply:
I'm wondering if users are thinking that PeerDB will create a table which will work with the query they give. This setting is more niche than it comes across. Maybe can introduce advanced settings for qrep and put this there

Reply:
Reverting this change for now. Can tackle it as a separate PR if needed